cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ansible-modules"]
+= Automating management tasks using Ansible
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-ansible-modules
+toc::[]
+
+Ansible is an automation tool used to configure systems, deploy software, and perform rolling updates.
+Ansible includes support for {ProductName}, and Ansible modules are available to allow you to automate
+cluster management tasks such as template, persistent volume claim and virtual machine operations.
+
+Ansible provides an alternative way of automating {ProductName} management compared to APIs and oc tool,
```

Comment:
Overall, the writing is good and you followed our conventions well (thanks for using {ProductName}! 

This particular sentence is a bit unclear to me, though. When you say it's "an alternative way" "compared to APIs and [the] `oc` tool," I'm not sure what is being compared. If you want to say that Ansible is better than using APIs or the `oc` tool, give a reason. :) Otherwise, I don't think it's necessary to mention APIs/the `oc` tool here.

Might be better to just say something like: "Ansible allows you to integrate with other Ansible modules, \<give an example or reason why you would want to do that\>."

Reply:
The intention was not to compare but state that it is yet another way to manage our `{ProductName}` but we have a benefit to use other modules provided by Ansible.